pkg/auditd/Makefile

```diff
@@ -0,0 +1,15 @@
+.PHONY: tag push
```

Comment:
can we use the new style Makefile?  Should be pretty simple, see https://github.com/linuxkit/linuxkit/blob/master/pkg/rngd/Makefile as an example